src/main/java/io/gatling/mojo/VerifyMojo.java

```diff
@@ -60,7 +59,7 @@ private void verifyLastRun() throws IOException, MojoFailureException, MojoExecu
   }
 
   private void checkError(String line) throws MojoFailureException {
-    if (StringUtils.contains(line, LAST_RUN_FILE_ERROR_LINE)) {
+    if (line != null && line.contains(LAST_RUN_FILE_ERROR_LINE)) {
```

Comment:
I think we can drop the null check as I really don't think `Files.readAllLines` can produce a null line, and if it were, that's something we'd rather have reported as a crash instead of being silently ignored.

Reply:
I agree. The PR is updated accordingly.